src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -171,6 +186,13 @@ export class TaskListWrapperComponent implements OnInit {
     };
   }
 
+  public getPaginationParameter(): PaginationParameter {
+    return {
+      page_size: this.pagination.page_size,
+      page_number: this.pagination.page_number
+    }
```

Comment:
Missing semicolon.

Reply:
You could also use the spread operator and make this simply:
`return { ...this.pagination };`

Reply:
Yeah I did it like this because it is what the other methods of similar vein do, will change to use spread anyway